.circleci/config.yml

```diff
@@ -705,3 +747,16 @@ jobs:
             export MAKE=mingw32-make
             $msys2 scripts/travis/build_test.sh
           shell: bash.exe
+  tests_verification_job:
+    docker:
+      - image: python:3.9.6-alpine
+    resource_class: medium
```

Comment:
could this be smaller?

Reply:
i'll check how much slower it is on small docker

Reply:
it is not slower, so switched to small, which is the smallest.